specification/iotcentral/resource-manager/Microsoft.IoTCentral/stable/2018-09-01/iotcentral.json

```diff
@@ -733,6 +774,52 @@
           "readOnly": true
         }
       }
+    },
+    "AppTemplate": {
+      "description": "IoT Central Application Template.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "The application template identifier.",
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
+          "description": "The application template name.",
+          "type": "string",
+          "readOnly": true
+        },
+        "properties": {
+          "description": "The extra template properties.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
```

Comment:
You should model all the properties so the generated SDK can take advantage of strong typing for languages that support it. In the example you have properties "manifestId", "manifestVersion", "appTemplateName", "title", "order", and "description". You should create a separate type called `AppTemplateProperties` and reference it here.